src/api/jobs.js

```diff
@@ -16,16 +16,14 @@ export async function getRunningJobs(accessionCode) {
   const [processorJobs, downloaderJobs] = await Promise.all([
     Ajax.get(`/v1/jobs/downloader/`, {
       sample_accession_code: accessionCode,
-      nomad: true,
     }),
     Ajax.get(`/v1/jobs/processor/`, {
       sample_accession_code: accessionCode,
-      nomad: true,
```

Comment:
This function no longer returns only running jobs, but instead all jobs with a field `is_queued` that indicates whether or not it's running.

I was about to suggest filtering this list based on `is_queued`, but I don't think the list of running jobs should be generated by making additional API call. Instead we should just look at the results of the `getDownloaderJobs()` and `getProcessorJobs()` and filter those based on `is_queued`.

Reply:
👍

Reply:
probably could remove this function actually but ill leave it in there for now/ever

Reply:
Is it still used? If possible it'd be ideal to avoid a separate API call.